packages/sanity/src/core/form/store/types/nodes.ts

```diff
@@ -13,23 +13,35 @@ import {FormNodePresence} from '../../../presence'
 import {ArrayOfObjectsMember, ArrayOfPrimitivesMember, ObjectMember} from './members'
 import {FormFieldGroup} from './fieldGroup'
 
-/** @public */
+/**
+ * @hidden
+ * @public
+ */
 export interface BaseFormNode<T = unknown, S extends SchemaType = SchemaType> {
   // constants
+  /** The unique identifier of the node. */
   id: string
+  /** The schema type of the node. */
   schemaType: S
+  /** The level of the node in the form hierarchy. */
   level: number
+  /** The path of the node in the form hierarchy. */
   path: Path
 
   // state
   /**
    * @hidden
    * @beta */
   presence: FormNodePresence[]
+  /** The validation errors of the node. */
```

Comment:
(validation can contain errors, warnings and info markers)

```suggestion
  /** The validation markers of the node. */
```